.github/workflows/linux.yml

```diff
@@ -388,6 +388,16 @@ jobs:
       container: '{"image": "openvinogithubactions.azurecr.io/dockerhub/ubuntu:20.04", "volumes": ["/mount:/mount"], "options": "-e SCCACHE_AZURE_BLOB_CONTAINER -e SCCACHE_AZURE_CONNECTION_STRING"}'
       sccache-azure-key-prefix: 'ubuntu20_x86_64_onnxruntime'
 
+  ONNX_Models:
```

Comment:
Is not added to the overall status job, for now, I suggest adding it there after we are sure it is working and when we delete the Azure workflow.